app/controllers/api_controller.rb

```diff
@@ -1,10 +1,12 @@
+require 'pry'
 class APIController < ActionController::API
   @@SLACK_VERIFICATION_TOKEN = ENV['SLACK_VERIFICATION_TOKEN']
   @@SLACK_CLIENT_ID = ENV['SLACK_CLIENT_ID']
   @@SLACK_CLIENT_SECRET = ENV['SLACK_CLIENT_SECRET']
 
   before_action :verify_token
   def verify_token
+    binding.pry
```

Comment:
might want to remove this before merging.

Reply:
You're right, I'll go through and get rid of all the places I put pry before merging